### ISSUE TITLE
Doc: missing step for renaming AppConstant.example.js

### ISSUE THREAD

**Issue by j1n6** [personium/personium-blank-app#34]
When running the application locally, the documentation was missing a step to rename:
```sh
mv src/app/frontend/Constants/AppConstant.example.js src/app/frontend/Constants/AppConstant.js
```

The `AppConstant` requires the following input:

```node
export const AppConstant = {
  cellUrl: '<APP_CELL_URL>',
  installBoxName: '<BOX_NAME>',
  barFileUrl: '<BAR_FILE_URL>',
};
```

**Comment by yoh1496** [comment]
I will modify README.md to show guide to set AppConstant.
Thank you.

**Comment by j1n6** [comment]
close this as it's been addressed in #36
